.github/dependabot.yml

```
@@ -15,6 +15,7 @@ updates:
       - "/pkg/apiserver"
       - "/pkg/build"
       - "/pkg/build/wire"
+      - "pkg/infra/log"
```

Are we missing a slash here?
```suggestion
      - "/pkg/infra/log"
```